plasmapy/analysis/tests/test_fit_functions.py

```diff
@@ -0,0 +1,517 @@
+"""
+Tests for the fitting function classes defined in `plasmapy.analysis.fit_functions`.
+"""
+import inspect
+import numpy as np
+import pytest
+
+from abc import ABC, abstractmethod
+
+import plasmapy.analysis.fit_functions as ffuncs
+
+
+class TestAbstractFitFunction:
+    """
+    Tests for fit function class `plasmapy.analysis.fit_functions.AbstractFitFunction`.
+    """
+
+    ff_class = ffuncs.AbstractFitFunction
+
+    def test_is_abs(self):
+        assert issubclass(self.ff_class, ABC)
+
+    def test_methods(self):
+        # required attributes/methods
```

Comment:
```suggestion
        """Test that `AbstractFitFunction` has the required methods and attributes."""
```
I tend to prefer using one-line docstrings for tests, since that often helps when I'm coming back to tests a few months later.